index.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en"> 
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://kit.fontawesome.com/e4fb871c71.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital@1&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+    <title>The BB Concert</title>
+</head>
+<body> 
+    <div class="top">
+        
+         <div class="dot">.</div>
+
+            <div class="others">
+            <i class="ele fa-brands fa-facebook-f"> </i>
+            <i class="ele fa-brands fa-twitter"> </i> 
+            <p class="ele">English</p> 
+            <p class="ele"> My_page </p> 
+            <p class="ele">logout</p>  
+           </div> 
+
+    </div>
+
+    <div class="Desktop-nav">  
+        <div class="logo"> 
+ <a href="index.html"><img id="the-logo" src="Images/the-logo.png" alt="pic"></a>
+        </div>
+
+        <div class="other-navigations">
+            <p class="elements">About</p>
+            <p class="elements">program</p>
+            <p class="elements">join</p>
+            <p class="elements">sponsor</p>
+            <p class="elements">News</p>
+            <div id="concert-name">
+                <p>BB</p><p> Concert</p> </div>
+        </div>
+
+    </div>
+     <div class="mens">
+        <div id="xmark-container">
+            <i class="fa-solid fa-x" id="x-mark"></i>
+         </div>
+
+        <div class="menu-line-container">
+            <p class="menu-line">.</p>
+        </div>
+         
+        <a href="index.html"><p>Home</p></a>  
+        <div class="menu-line-container">
+            <p class="menu-line">.</p>
+        </div>
+       <a href="about.html"><p>About</p></a> 
+    </div>
+
+    <section class="main-page" id="main-page-desktop">
+        <h1>.</h1>
```

Comment:
Please be sure to remove this unnecessary dot from your website so that your design resembles as much as possible the given design 🚀 You could apply the `h1` tag to your "Yaaa! Let's Light Up The Stage" paragraph tag 🧐

Reply:
DONE

Reply:
i place in the dot to pass the linter.